game/scripts/npc/abilities/enigma_black_hole.txt

```diff
@@ -24,7 +24,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "200.0 180.0 160.0 145.0 130.0"
+    "AbilityCooldown"                                     "140.0" //OAA
```

Comment:
AAAAAAAAAAA for the love of god. Set the cooldown to 160 at all levels or don't change it at all.